ietf-microwave-radio-link.yang

```diff
@@ -389,30 +489,20 @@ module ietf-microwave-radio-link {
         reference
           "ETSI EN 302 217-1";
         leaf maximum-nominal-power {
-          type mw-types:power {
+          type power {
             range "-99..99";
           }
           units "dBm";
           mandatory true;
           description
-            "Nominal maximum output power.";
+            "Selected maximum output power.  Minimum output
+             power is the same as the system capability,
+             minimum-power.";
           reference
             "ETSI EN 302 217-1";
         }
-        leaf minimum-nominal-power {
-          type mw-types:power {
-            range "-99..99";
-          }
-          units "dBm";
-          mandatory true;
-          description
-            "Minimum output power.
-
-             By default, minimum output power is the same as the
-             system capability minimum-power.";
-        }
```

Comment:
Why the attributes for ATCP have been changed in this PR (which is intended to add only header compression attributes)?

I am wondering whether some of the previous changes to the main branch got lost